Formula/globjects.rb

```diff
@@ -0,0 +1,34 @@
+class Globjects < Formula
+  desc "C++ library strictly wrapping OpenGL objects"
+  homepage "https://github.com/cginternals/globjects"
+  url "https://github.com/cginternals/globjects/archive/v1.0.0.tar.gz"
+  sha256 "be2f95b4e98eef61a57925985735af266fef667eec63a39f65def5d5d808a30a"
+  head "https://github.com/cginternals/globjects.git"
+
+  depends_on "cmake" => :build
+  depends_on "glbinding"
+
+  needs :cxx11
+  def install
+    args = []
+
+    args << "-Dglbinding_DIR=" + Formula["glbinding"].prefix
```

Comment:
You don’t need to build an array here; just add `"-Dglbinding_DIR=#{Formula["glbinding"].opt_prefix}"` to the arguments list below.